system_tests/monitoring.py

```diff
@@ -181,6 +183,53 @@ def test_create_and_delete_metric_descriptor(self):
 
         retry_500(descriptor.create)()
         retry_404_500(descriptor.delete)()
+        descriptor.create()
+        retry_404(descriptor.delete)()
+
+        with self.assertRaises(NotFound):
+            descriptor.delete()
+
+    def test_write_point(self):
+        METRIC_TYPE = ('custom.googleapis.com/tmp/system_test_example' +
+                       unique_resource_id())
+        METRIC_KIND = monitoring.MetricKind.GAUGE
+        VALUE_TYPE = monitoring.ValueType.DOUBLE
+        DESCRIPTION = 'System test example -- DELETE ME!'
+        VALUE = 3.14
+
+        client = monitoring.Client()
+        descriptor = client.metric_descriptor(
+            METRIC_TYPE,
+            metric_kind=METRIC_KIND,
+            value_type=VALUE_TYPE,
+            description=DESCRIPTION,
+        )
+
+        descriptor.create()
+        retry_404(descriptor._fetch)(client, METRIC_TYPE)
```

Comment:
Ah sorry I did miss this comment but I'm confused why you would prefer to replace it with time.sleep(). I think the retry decorators are a little better since they can use less time sometimes and are less flaky with the exponential backoff. 

I think if anything you're write I should get rid of this then retry write_point? That makes sense to me